integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -0,0 +1,445 @@
+					],
+					"path": [
+						"api",
+						"v1",
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Create a course",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "lt5H1qDZfNo9aglt7jb9fLCLdRb1cEZNMdD3VZjORE3o82jyV1LAxNVI0up5fnur",
+							"type": "string"
+						}
+					]
+				},
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "course[name]",
+							"value": "binod created a course using form data",
+							"type": "text"
+						}
+					],
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/accounts/2/courses",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"accounts",
+						"2",
+						"courses"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "DELETE a course",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "lt5H1qDZfNo9aglt7jb9fLCLdRb1cEZNMdD3VZjORE3o82jyV1LAxNVI0up5fnur",
+							"type": "string"
+						}
+					]
+				},
+				"method": "DELETE",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "course[name]",
+							"value": "binod created a course using form data",
+							"type": "text"
+						}
+					],
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses/<<id>>?event=delete",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"courses",
+						"<<id>>"
+					],
+					"query": [
+						{
+							"key": "event",
+							"value": "delete"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET course settings",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": ""
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Update course settings",
+			"request": {
+				"method": "PUT",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/api/v1/courses/13/settings",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"api",
+						"v1",
+						"courses",
+						"13",
+						"settings"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Update a course",
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "lt5H1qDZfNo9aglt7jb9fLCLdRb1cEZNMdD3VZjORE3o82jyV1LAxNVI0up5fnur",
+							"type": "string"
+						}
+					]
+				},
+				"method": "PUT",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "course[image_url]",
+							"value": "lt5H1qDZfNo9aglt7jb9fLCLdRb1cEZNMdD3VZjORE3o82jyV1LAxNVI0up5fnur",
```

Comment:
why is the image url the same as the token? lol.
Given you have access_token as a var, I'd replace all spots with the token's value with the variable `{{access_token}}`

Reply:
haha fixed it